detox/src/utils/customConsoleLogger.js

```diff
@@ -0,0 +1,56 @@
+const util = require('util');
+const path = require('path');
+const callsites = require('./callsites');
+
+function callsitesToStack() {
+  return callsites().slice(2).reduce((result, callsite) => {
```

Comment:
Why can't we use something like `new Error().stack` for that?

Reply:
This in essence what callsites does!

Reply:
I was referring to a complex error string formatting logic, this is not what callsites does.

Reply:
ok, you got me - I got too excited about using callsites so I abused it a bit. 'Changed it now as per your suggestion :D